securesystemslib/rsa_keys.py

```diff
@@ -473,9 +481,18 @@ def verify_rsa_signature(signature, signature_scheme, public_key, data):
     # hashing algorithm.
     try:
       if signature_scheme.startswith('rsassa-pss'):
+        # https://github.com/golang/go/blob/11f92e9dae96939c2d784ae963fa7763c300660b/src/crypto/rsa/pss.go#L269-L275
```

Comment:
Can you add a sentence or two, why you add this link to Go code here?
Also what is `padding.PSS.MAX_LENGTH`? For me, it sounds like that the padding determines the pss salt length.
Is this correct?